prover/prover_dal/src/fri_prover_dal.rs

```diff
@@ -754,6 +754,27 @@ impl FriProverDal<'_, '_> {
             .await
     }
 
+    pub async fn delete_batch_data_for_aggregation_round(
+        &mut self,
+        l1_batch_number: L1BatchNumber,
+        aggregation_round: AggregationRound,
+    ) -> sqlx::Result<sqlx::postgres::PgQueryResult> {
```

Comment:
nit: I don't really like returning the PgQueryResult, can't we map the error right here if any?